app/helpers/application_helper.rb

```diff
@@ -50,4 +50,16 @@ def format_currency(number, options = {})
 
     number_to_currency(number, options)
   end
+
+  def nav_link(text, path, icon, current_page)
+    classes = 'block border border-transparent rounded-xl p-2 text-sm font-medium text-gray-500 flex items-center'
+    classes += ' hover:bg-white hover:border-[#141414]/[0.07] hover:text-gray-900 hover:shadow-xs' unless current_page?(path)
+  
+    content_tag(:li) do
+      link_to(path, class: classes) do
```

Comment:
```suggestion
    classes = 'block border border-transparent rounded-xl p-2 text-sm font-medium text-gray-500 flex items-center'
    selected_classes = 'hover:bg-white hover:border-[#141414]/[0.07] hover:text-gray-900 hover:shadow-xs'
  
    content_tag(:li) do
      link_to(path, class: class_list(classes, selected_classes: current_page?(path)) do
```

Reply:
note: `current_page` is already passed in as a boolean so I don't think you need the `current_path?(path)` test again?

Reply:
Done👍🏻